[2024-04-13 19:03:30,708][urllib3.connectionpool][WARNING] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x1006a5180>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/4504800232407040/envelope/
[2024-04-13 19:03:30,709][urllib3.connectionpool][WARNING] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x1006a5120>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/4504800232407040/envelope/
[2024-04-13 19:03:30,710][urllib3.connectionpool][WARNING] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x1006a5000>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/4504800232407040/envelope/
[2024-04-13 19:03:30,712][urllib3.connectionpool][WARNING] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x1006a5f30>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/4504800232407040/envelope/
[2024-04-13 19:03:30,712][urllib3.connectionpool][WARNING] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x1006a5810>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/4504800232407040/envelope/
[2024-04-13 19:03:30,713][urllib3.connectionpool][WARNING] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x1006a53f0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known')': /api/4504800232407040/envelope/
